compose.yaml

```
@@ -0,0 +1,9 @@
+services:
+  db:
+    image: "postgis/postgis:15-3.4-alpine"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=zoning
+    ports:
+      - "5432:5432"
```

I think we'll quickly want to add a volume to preserve work across docker containers

```suggestion
      - "5432:5432"
      volumes:
      - ./db-volume:/var/lib/postgresql/data
```